arch/risc-v/include/setjmp.h

```diff
@@ -25,13 +25,25 @@
  * Included Files
  ****************************************************************************/
 
+#include <nuttx/config.h>
+#include <nuttx/compiler.h>
+
 /****************************************************************************
  * Public Types
  ****************************************************************************/
 
 struct setjmp_buf_s
 {
   long regs[14];
+
+  /* Float callee register : fs0-fs11 */
+#ifdef CONFIG_ARCH_DPFPU
```

Comment:
should we move CONFIG_ARCH_DPFPU to first?

Reply:
Emm, it's typo, it should be CONFIG_ARCH_QPFPU